Author: Andres Mejia <mcitadel@gmail.com>
Date: Wed Jun 2 18:30:20 2010 -0400
Description: Patch to remove unnecessary dependency to libva-x11 for dummy driver

==========================================================================
--- a/dummy_drv_video/Makefile.am
+++ b/dummy_drv_video/Makefile.am
@@ -26,8 +26,6 @@
 dummy_drv_video_la_LTLIBRARIES	= dummy_drv_video.la
 dummy_drv_video_ladir		= $(LIBVA_DRIVERS_PATH)
 dummy_drv_video_la_LDFLAGS	= -module -avoid-version -no-undefined -Wl,--no-undefined
-dummy_drv_video_la_LIBADD	= $(top_builddir)/va/$(libvabackendlib)
-dummy_drv_video_la_DEPENDENCIES	= $(top_builddir)/va/$(libvabackendlib)
 dummy_drv_video_la_SOURCES	= dummy_drv_video.c object_heap.c
 noinst_HEADERS			= dummy_drv_video.h object_heap.h
 endif
